languagetool-standalone/src/main/java/org/languagetool/gui/RightClickListener.java

```diff
@@ -0,0 +1,39 @@
+/* LanguageTool, a natural language style checker 
+ * Copyright (C) 2016 Daniel Naber (http://www.danielnaber.de)
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+package org.languagetool.gui;
+
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+class RightClickListener extends MouseAdapter {
+  public void mousePressed(MouseEvent e) {
+    if (e.isPopupTrigger())
```

Comment:
Please add {} also for `if` blocks with only one statement.

Reply:
ok